include/mbedtls/asn1write.h

```diff
@@ -54,8 +54,7 @@ int mbedtls_asn1_write_len( unsigned char **p, unsigned char *start, size_t len
  *
  * \return          the length written or a negative error code
  */
-int mbedtls_asn1_write_tag( unsigned char **p, unsigned char *start,
-                    unsigned char tag );
+int mbedtls_asn1_write_tag( unsigned char **p, unsigned char *start, int tag );
```

Comment:
This is a breaking change to the interface unfortunately. To introduce the functionality I think you want, you'll need to create a new function.